payloads/v10/_interactions/modalSubmit.ts

```diff
@@ -17,6 +17,10 @@ export interface ModalSubmitActionRowComponent extends APIBaseComponent<Componen
 	components: ModalSubmitComponent[];
 }
 
+export interface ModalSubmitLabelComponent extends APIBaseComponent<ComponentType.Label> {
+	component: APIComponentInModalLabel;
```

Comment:
This type is still wrong, and its a ModalSubmitComponent with values instead of value! We'd wanna turn ModalSubmitComponent into a union of strictly typed components (and also add in missing fields like `id` lols)